packages/shared/styles/components/SfSteps.scss

```diff
@@ -0,0 +1,103 @@
+@import "../variables";
+
+
+$sf-steps__active-line-height-mobile: 0.1875rem;
+$sf-steps__active-line-height-desktop: 0.125rem;
+$sf-steps__active-line-color: #1d1f22;
```

Comment:
do we have any global modifier for this active color already?

Reply:
I'll check